normandy/recipes/tests/test_api.py

```diff
@@ -208,12 +211,14 @@ def test_it_can_enable_recipes(self, api_client):
         recipe = Recipe.objects.all()[0]
         assert recipe.enabled
 
+    @pytest.mark.xfail
```

Comment:
`xfail` takes an optional `reason` kwarg. I think we should use that for all xfails.